account_constraints/model/account_move_line.py

```diff
@@ -28,7 +28,8 @@ class AccountMoveLine(models.Model):
     def _get_write_authorized_fields(self):
         """ This method can be overrride to add some field that can be written
         directly on account move line """
-        return ["reconcile_id", "reconcile_partial_id"]
+        return ["reconcile_id", "reconcile_partial_id", "followup_line_id",
```

Comment:
@pedrobaeza these fields are part of the official account_followup.

The most correct solution would be an auto_install module that glues account_followup and account_constraints.

But I'm not shocked with the solution proposed by @adrienpeiffer